moveit_ros/planning/planning_scene_monitor/src/planning_scene_monitor.cpp

```diff
@@ -171,6 +177,7 @@ planning_scene_monitor::PlanningSceneMonitor::~PlanningSceneMonitor()
   parent_scene_.reset();
   robot_model_.reset();
   rm_loader_.reset();
+  spinner_.stop();
```

Comment:
Is it well defined what happens if this was never started?

Reply:
Yes. AsyncSpinner just doesn't do anything:
http://docs.ros.org/melodic/api/roscpp/html/spinner_8cpp_source.html#l00289